src/Maestro/DependencyUpdater/DependencyUpdater.cs

```diff
@@ -203,6 +207,63 @@ select new
             }
         }
 
+        /// <summary>
+        ///     Update Longest Build Path table once a day for each channel at midnight
+        /// </summary>
+        /// <param name="cancellationToken"></param>
+        /// <returns></returns>
+        [CronSchedule("0 0 0 1/1 * ? *", TimeZones.PST)]
+        public async Task UpdateLongestBuildPathAsync(CancellationToken cancellationToken)
+        {
+            IQueryable<Channel> query = Context.Channels;
+
+            List<Channel> channels = query.AsEnumerable().Select(c => new Channel() { Id = c.Id, Name = c.Name }).ToList();
+
+            // Get the flow graph
+            var barOnlyRemote = await RemoteFactory.GetBarOnlyRemoteAsync(Logger);
+
+            List<Microsoft.DotNet.Maestro.Client.Models.DefaultChannel> defaultChannels = (await barOnlyRemote.GetDefaultChannelsAsync()).ToList();
+            List<Microsoft.DotNet.Maestro.Client.Models.Subscription> subscriptions = (await barOnlyRemote.GetSubscriptionsAsync()).ToList();
+
+            IEnumerable<string> frequencies = 
+                new string[] { "everyWeek", "twiceDaily", "everyDay", "everyBuild", "none", };
+
+            foreach (var channel in channels)
+            {
+                // Build, then prune out what we don't want to see if the user specified
+                // channels.
```

Comment:
>// channels. [](start = 16, length = 12)

nit... probably wouldn't wrap this single word... especially since the next line is longer. :-)